publisher.go

```diff
@@ -82,6 +84,9 @@ func (p *Publisher) ExchangeDeclare(exchange string) error {
 }
 
 func (p *Publisher) Publish(params *PublishParams) error {
+	fmt.Println("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA")
```

Comment:
I assume you forgot to remove the "AAAAAAAAAAAAAAAAA" log?

Reply:
🙈